### DISCUSSION TITLE
How do I convert my project from Mac to Linux?

### DISCUSSION THREAD

**Discussion by MattHamp21** [pocketbase/pocketbase#2137] [Q&A]
I made a next.js project using pocketbase on my mac. Now I'm trying to run my project on a LInux VM. I tired switching just the MacOs version for the Linux version but I kept running into issues. Is there any easy way to make the switch painless?

**Comment by ganigeorgiev** [comment]
What issues did you run into?

When switching between platforms, it should be enough to just download and run the correct binary for your platform.
